src/Elastic.Apm/Model/ExecutionSegmentCommon.cs

```diff
@@ -164,6 +165,46 @@ internal static Task<T> CaptureSpan<T>(Span span, Func<ISpan, Task<T>> func)
 			});
 		}
 
+		private const string DefaultCulprit = "PublicAPI-CaptureException";
+		private static string GetCulprit(Exception exception, IConfigurationReader configurationReader)
+		{
+			if (exception == null) return DefaultCulprit;
+
+			var stackTrace = new StackTrace(exception);
+			var frames = stackTrace.GetFrames();
+			if(frames == null) return DefaultCulprit;
+
+			var excludedModules = configurationReader.ExcludedNamespaces;
+
+			foreach (var frame in frames)
+			{
+				var method = frame.GetMethod();
+				var module = method.DeclaringType?.FullName ?? "Unknown Module";
```

Comment:
nit: Wouldn't be `fullyQualifiedTypeName` a better name for this variable?  - since that's what it is, it's not really the module. Also I'd rename the parameter name of `IsInApp`.